pkg/nanopb/Makefile.gensrc

```diff
@@ -2,29 +2,40 @@ PROTOC ?= protoc
 PROTOC_GEN_NANOPB ?= $(PKGDIRBASE)/nanopb/generator/protoc-gen-nanopb
 
 PROTOBUF_FILES ?= $(wildcard *.proto)
```

Comment:
In our branch we have 

```suggestion
PROTOBUF_FILES += $(wildcard *.proto)
```

and apps that don't build with the upstream version.